error-handling/src/main/java/com/palantir/remoting2/errors/SerializableStackTraceElement.java

```diff
@@ -30,13 +30,23 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public abstract class SerializableStackTraceElement implements Serializable {
 
-    public abstract Optional<String> getClassName();
+    public abstract String getClassName();
 
-    public abstract Optional<String> getMethodName();
+    public abstract String getMethodName();
 
     public abstract Optional<String> getFileName();
 
-    public abstract Optional<Integer> getLineNumber();
+    public abstract int getLineNumber();
+
+    @Override
+    public final String toString() {
+        StackTraceElement element = new StackTraceElement(getClassName(),
```

Comment:
please add a test verifying that this works when class and method are invalid java identifiers, i.e., 'Bogus Class Name', and 'bogus method name'.